index.html

```diff
@@ -0,0 +1,239 @@
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cum fugit perspiciatis doloremque quis ratione, natus quaerat inventore illo! Quo dolores numquam eveniet illum porro magnam nisi doloremque vitae minima praesentium?</p>
+                    <img src="https://www.computersciencedegreehub.com/wp-content/uploads/2016/02/what-is-coding-1024x683.jpg" alt="">
+                </div>
+                <div class="feed__icons">
+                    <span><img class="align" src="images/reply.svg" alt=""> 245</span>
+                    <span><img class="align" src="images/retweet.svg" alt=""> 932</span>
+                    <span><img class="align" src="images/like.svg" alt=""> 1k</span>
+                    <span><img class="align" src="images/share.svg" alt=""></span>
+                </div>
+            </div>
+                <div class="feed__element">
+                <div class="feed__profile"><img src="https://www.hakuhodo-global.com/wp_admin/wp-content/uploads/2017/11/Kazuhiro_Suda2square.jpg" alt=""></div>
+                <div class="feed__main">
+                    <h4>Adam Knuessberg</h4><img class="verified align" src="/images/verified.svg" alt=""><span>@AKnuess · 34m</span>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cum fugit perspiciatis doloremque quis ratione, natus quaerat inventore illo! Quo dolores numquam eveniet illum porro magnam nisi doloremque vitae minima praesentium?</p>
+                    <img src="images/feed-image.PNG" alt="">
+                </div>
+                <div class="feed__icons">
+                    <span><img class="align" src="images/reply.svg" alt="" > 245</span>
+                    <span><img class="align" src="images/retweet.svg" alt="" > 932</span>
+                    <span><img class="align" src="images/like.svg" alt="" > 1k</span>
+                    <span><img class="align" src="images/share.svg" alt="" ></span>
+                </div>
+            </div>
+                <div class="feed__element">
+                <div class="feed__profile"><img src="https://yt3.ggpht.com/ytc/AKedOLSRSl8xsTNuQU_f6sg3bHI19gZYUSqLu2I78S90MQ=s900-c-k-c0x00ffffff-no-rj" alt=""></div>
+                <div class="feed__main">
+                    <h4>bob34h7</h4><span> @bob34h7 · 2h</span>
+                    <p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Cum fugit perspiciatis doloremque quis ratione, natus quaerat inventore illo! Quo dolores numquam eveniet illum porro magnam nisi doloremque vitae minima praesentium?</p>
+                    <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ex sequi provident nemo repellat culpa laudantium unde praesentium illo quod, exercitationem voluptas eum omnis corporis nisi in earum obcaecati, adipisci dolor!</p>
+                </div>
+                <div class="feed__icons">
+                    <span><img class="align" src="images/reply.svg" alt="" > 245</span>
```

Comment:
Might want to have the `alt` property filled in for these. It really helps with screen-readers and other accessibility devices.